### PR TITLE
bext: Fix SG button in file page header

### PR DESCRIPTION
**Pull request by fkling**
For reasons unknown I looked at the browser extension. This seems to fix the issues of showing the "open in sourcegraph" button in the file header.

I modified the E2E test to make hovercard testing optional so that we can at least test the toolbar button.

Addendum: I also updated the pipline generate to run browser extension tests when code inside `client/browser/` changes.

## Test plan

Manual testing and reenabling the E2E test for the button specifically.

### CODE REVIEW

**Review by vovakulikov — APPROVED** [review]

**Comment by peterguy** [comment]
@felixfbecker the reasons may have been unknown to you, but they were [serendipitous](https://sourcegraph.slack.com/archives/C022SPMNR0W/p1697499310532359)

**Comment by felixfbecker** [comment]
@peterguy I no longer have access to click that link 😄

**Comment by peterguy** [comment]
My apologies @felixfbecker! 🤦 I meant to tag @fkling , but GH auto-filled your handle and I didn't notice.

File: dev/ci/internal/ci/pipeline.go [hunk 1]
```diff
@@ -159,6 +159,14 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 				triggerAsync(buildOptions)))
 		}
 
+		if c.Diff.Has(changed.ClientBrowserExtensions) {
+			ops.Merge(operations.NewNamedSet("Browser Extensions",
```
Comment (fkling) [inline dev/ci/internal/ci/pipeline.go:163]: I didn't know what the "name set" does, now I do. I guess it's not necessary to group them? Idk what best practices here are...

Comment (jhchabran) [reply dev/ci/internal/ci/pipeline.go:163]: @fkling it creates a "group" of jobs in CI, like the ones that publish images, or the security jobs. It makes sense to have one if you have a few jobs inside there, which is the case here. 

Are you sure you need the "frontend-tests" though ?

Comment (fkling) [reply dev/ci/internal/ci/pipeline.go:163]: > Are you sure you need the "frontend-tests" though ?

I guess those should be covered by other operations.

Comment (fkling) [reply dev/ci/internal/ci/pipeline.go:163]: @jhchabran Is this g2m (good to merge) from your side then?

**Pull request merged**